defaults/main.yml

```diff
@@ -5,6 +5,10 @@
 sap_hana_deployment_bundle_path:
 sap_hana_deployment_bundle_sar_file_name:
 
+# Target host path and file name for the zip installation file
+sap_hana_deployment_zip_path:
+sap_hana_deployment_zip_file_name:
+
```

Comment:
I can't see any reference in the README to these variables, can you please add this?